airbyte-webapp-e2e-tests/cypress/plugins/index.ts

```diff
@@ -14,13 +14,25 @@
 
 import Cypress from "cypress";
 
+const pgp = require("pg-promise")();
+const postgresConfig = require(require("path").resolve("cypress.json"));
+
+function dbConnection(query: any, userDefineConnection: undefined) {
```

Comment:
Is `undefined` the correct type for this?

Reply:
Hm... Missed that! Will fix that! Thanks!